README.md

```diff
@@ -31,3 +31,25 @@ SELECT * FROM posts OFFSET 20 LIMIT 11;
 ```
 
 it fetches one more record (11 = 10 + 1) to calculate if there is a next page records, so it doesn't need to send COUNT sql.
+
+
+### Usage
+
+```
+## perform a paginate query:
+@users = User.paginate(:page => params[:page])
+
+## render previous page link:
+<%= link_to_previous_page @users, 'Previous' %>
+
+## render next page link:
+<%= link_to_next_page @users, 'Next' %>
```

Comment:
why not use something like `<%= simple_paginate @users %>`, just like `<%= paginate @users %>` in kaminari